main/src/library/Prelude.flix

```diff
@@ -206,15 +206,17 @@ pub def touch(_: Region[r]): Unit \ r = checked_ecast(())
 ///
 /// Crashes the current process with the given message `m`.
 ///
-pub def bug!(m: String): a = unchecked_cast({
+pub def bug!(m: String): a = unchecked_cast({ printBugMsg(m); ?panic } as _ \ {})
+
+@Internal
+pub def printBugMsg(m: String): Unit \ IO = {
```

Comment:
This can just be private right?

Reply:
We are at the prelude

Reply:
Right, would have to add a mod to hide it